examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -184,7 +193,6 @@ def _save_forcefield(filename, ff_params):
         prod_steps=configuration.num_prod_steps,
     )
 
-    # TODO: use binding_model.predict rather than binding_model.loss
```

Comment:
@maxentile Based on your last changes I think this comment is no longer relevant?

Reply:
Good catch -- thank you for spotting and correcting that!